### ISSUE TITLE
improve colors and contrast in dark theme 

### ISSUE THREAD

**Issue by jolesbi** [cosmos/voyager#763]
we're still violating some accessibility standards in our dark theme. plus, i think the dark theme can be made more slick and readable.

i will propose a few colour changes 🎨

**Comment by nylira** [comment]
I support improving usability. These are our official brand colors though. Making any changes in the Voyager theme means we have to integrate the same changes in github.com/cosmos/cosmos.network and github.com/cosmos/explorer. This is another point in favor of #696.

**Comment by jolesbi** [comment]
i will definitely keep this in mind - i will try not to introduce new colours but rather see if i can rearrange them.
